package.json

```diff
@@ -51,6 +51,7 @@
         "@nestjs/platform-express": "^8.0.0",
         "@nestjs/serve-static": "^2.2.2",
         "@nestjs/swagger": "6.1.2",
+        "@reduxjs/toolkit": "^1.8.6",
```

Comment:
Make this change on main then bubble it through before merging down. You might want to just reset package.json and package-lock.json for now on this branch.